libr/core/cmd_debug.c

```diff
@@ -3879,18 +3879,21 @@ static bool cmd_dcu (RCore *core, const char *input) {
 }
 
 static int cmd_debug_continue (RCore *core, const char *input) {
-	int pid, old_pid, signum;
+	int pid, old_pid, signum, ret = true;
 	char *ptr;
 	// TODO: we must use this for step 'ds' too maybe...
 	switch (input[1]) {
 	case 0: // "dc"
 		r_reg_arena_swap (core->dbg->reg, true);
 #if __linux__
 		core->dbg->continue_all_threads = true;
-		r_debug_continue (core->dbg);
+		ret = r_debug_continue (core->dbg);
 #else
 		r_debug_continue (core->dbg);
```

Comment:
It seems weird to have twice `r_debug_continue`... Are you saying that the return value differs according to the OS?
Imho `ret = r_debug_continue` should be right after the `#endif`.

Reply:
> Are you saying that the return value differs according to the OS?

No. If it differs it's a bug since the code must abstract out any OS differences. The code was like that since testing was only done on Linux.